hack/testing/test-upgrade.sh

```diff
@@ -243,6 +243,11 @@ function verifyUpgrade() {
   local checkMigrate=${2:-false}
   local checkCDMMigrate=${3:-false}
 
+### check min_master migration
```

Comment:
you will also need to create a function that will remove the `MIN_MASTER` entry before running the upgrade, otherwise this will pass simply because a new install provides it